src/app/components/MessageBanner/index.tsx

```diff
@@ -0,0 +1,86 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import React, { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Heading from '../Heading';
+import Image from '../Image';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import styles from './index.styles';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+interface MessageBanner {
+  summaries: {
+    imageUrl?: string;
+    link: string;
+    imageAlt: string;
+    description?: string;
+    title: string;
+    id: string;
+  }[];
+  title: string;
+}
+const MessageBanner = ({ summaries, title }: MessageBanner) => {
+  const { dir } = useContext(ServiceContext);
+  const isRtl = dir === 'rtl';
+
+  return (
+    <section
+      css={styles.container}
+      role="region"
+      aria-labelledby="message-banner"
+    >
+      <div css={styles.card}>
+        {summaries.map(summary => {
+          return (
+            <React.Fragment key={summary.id}>
+              <div css={styles.textWrap}>
+                <Heading
+                  level={2}
+                  size="paragon"
+                  css={styles.heading}
+                  id="message-banner"
+                >
+                  {title}
+                </Heading>
+                <Paragraph size="longPrimer" css={styles.paragraph}>
+                  {summary.description}
+                </Paragraph>
```

Comment:
Would we still see an empty div with the paragraph css if there was no description? I am just wondering, since description is optional, should we add a check here (like we do for the image on line 64)

Reply:
It does not appear to be possible to have an empty promo summary in Tipo, nor overtype with an empty string. So description will always be available / visible and you can ignore this comment!